ProjectTodo/src/containers/HomeScreen.js

```diff
@@ -57,30 +82,50 @@ export default class HomeScreen extends React.Component {
 
   render() {
     cardArray = [];
+    cardDoneArray = [];
+
     for (var index in this.state.cards) {
-      cardArray.push({
-        key: this.state.cards[index].id + '',
-        content: this.state.cards[index].content,
-        title: this.state.cards[index].title
-      });
+      if (this.state.cards[index].done) {
```

Comment:
Render metoda sluzi samo za renderovanje nista vise! 
tako da ovo mora u neku drugu funkciju...
Takodje koristi foreach umesto for petlje